workitem/workitem_repository.go

```diff
@@ -278,3 +278,27 @@ func (r *GormWorkItemRepository) List(ctx context.Context, criteria criteria.Exp
 
 	return res, count, nil
 }
+
+// CheckWorkItemExists returns nil if no work item ID string is given or if work
+// item ID string could be converted into a number and looked up in the
+// database; otherwise it returns an error.
+func CheckWorkItemExists(db *gorm.DB, wiIDStr string) (*WorkItem, error) {
```

Comment:
I found I can use WorkItemRepository.LoadFromDB instead. It does the same thing.